src/app/client/src/app/modules/workspace/workspace.module.ts

```diff
@@ -44,7 +44,7 @@ import { MatTooltipModule } from '@angular/material/tooltip';
     OrderModule,
     PlayerHelperModule,
     ContentSearchModule,
-    CollectionEditorLibraryModule,
+    // CollectionEditorLibraryModule,
```

Comment:
@princegupta1131 
removed commented code if not required

Reply:
Removed the commeted line